clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/bam/BamMetadata.scala

```diff
@@ -0,0 +1,58 @@
+package org.broadinstitute.clio.transfer.model.bam
+
+import java.net.URI
+
+import org.broadinstitute.clio.transfer.model.{DeliverableMetadata, Metadata}
+import org.broadinstitute.clio.util.model.DocumentStatus
+
+case class BamMetadata(
```

Comment:
No metrics files?

Reply:
We agreed no metrics files at this time.